packages/experience-builder-sdk/src/core/componentRegistry.ts

```diff
@@ -160,7 +160,9 @@ export const optionalBuiltInComponents = [
 
 export const sendRegisteredComponentsMessage = () => {
   // Send the definitions (without components) via the connection message to the experience builder
-  const registeredDefinitions = Array.from(componentRegistry.values());
+  const registeredDefinitions = Array.from(componentRegistry.values()).map(
+    ({ definition }) => definition,
+  );
 
```

Comment:
We used it to support the case when customers would have a need to register components dynamically. It will send REGISTERED_COMPONENTS event instead of CONNECTED event

Reply:
So I guess that since we have never tried to register components dynamically, we never hit this issue